src/data/db2_secondHalfAppData.js

```diff
@@ -0,0 +1,484 @@
+export const db2_secondHalfAppData = {
```

Comment:
Improve variable and file names?

Reply:
I removed this file, and created a separate file for each of the `mock users`.